recipes/googleapis/all/conanfile.py

```diff
@@ -65,25 +68,10 @@ def validate(self):
     def requirements(self):
         self.requires('protobuf/3.21.4')
 
-    @property
-    def _cmake_new_enough(self):
-        try:
-            import re
-            output = StringIO()
-            self.run("cmake --version", output=output)
-            m = re.search(r'cmake version (\d+)\.(\d+)\.(\d+)', output.getvalue())
-            major, minor = int(m.group(1)), int(m.group(2))
-            assert major >= 3 and minor >= 20
-        except:
-            return False
-        else:
-            return True
-
     def build_requirements(self):
         self.build_requires('protobuf/3.21.4')
         # CMake >= 3.20 is required. There is a proto with dots in the name 'k8s.min.proto' and CMake fails to generate project files
-        if not self._cmake_new_enough:
-            self.build_requires('cmake/3.23.2')
+        self.build_requires('cmake/[>3.19]')
```

Comment:
Ranged version is not allowed in CCI https://github.com/conan-io/conan-center-index/blob/c16725e7abdab63935ee8607d59d1ae49b3a744a/docs/adding_packages/dependencies.md#version-ranges

Please, revert it.

Reply:
Done